gtsam/discrete/discrete.i

```diff
@@ -92,12 +92,28 @@ class DiscreteBayesNet {
 };
 
 #include <gtsam/discrete/DiscreteBayesTree.h>
+class DiscreteBayesTreeClique {
+  DiscreteBayesTreeClique();
+  DiscreteBayesTreeClique(const gtsam::DiscreteConditional* conditional);
+  const gtsam::DiscreteConditional* conditional() const;
+  bool isRoot() const;
+  void printSignature(
+      const string& s = "Clique: ",
+      const gtsam::KeyFormatter& formatter = gtsam::DefaultKeyFormatter) const;
```

Comment:
newline before bracket

Reply:
actually, tried re-formatting, stays the same.